corehq/apps/domain/auth.py

```diff
@@ -225,7 +225,7 @@ def get_active_users_by_email(email):
             yield user
         else:
             # also any mobile workers from TWO_STAGE_USER_PROVISIONING domains should be included
-            couch_user = CouchUser.get_by_username(user.username)
+            couch_user = CouchUser.get_by_username(user.username, strict=True)
             if couch_user.is_commcare_user() and TWO_STAGE_USER_PROVISIONING.enabled(couch_user.domain):
```

Comment:
```suggestion
            if couch_user and couch_user.is_commcare_user() and TWO_STAGE_USER_PROVISIONING.enabled(couch_user.domain):
```

Reply:
nvm, looks like this was independently added in https://github.com/dimagi/commcare-hq/pull/27758/